gitbootcamp.rst

```diff
@@ -107,7 +107,7 @@ Create a new branch and switch to it::
 
 This is equivalent to::
 
-   # create a new branch off 'master', without checking it out
+   # create a new branch off master, without checking it out
```

Comment:
I agree with removing the quotations, but I think this provides a further improvement to the wording:
```suggestion
   # create a new branch from master, without checking it out
```

Reply:
I agree.